installbuilder/datafiles/Base_DSC.data

```diff
@@ -254,10 +259,15 @@ ln -fs $LIBCURL_SO /opt/omi/lib/libcurl.so.4
 
 #if BUILD_OMS == 1
 su - omsagent -c "/opt/microsoft/${{SHORT_NAME}}/Scripts/RegenerateInitFiles.py"
+su - omsagent -c "/opt/microsoft/${{SHORT_NAME}}/Scripts/ImportGPGKey.sh /opt/microsoft/${{SHORT_NAME}}/keys/msgpgkey.asc keymgmtring.gpg"
```

Comment:
There is already code in importGPGkey.sh that deals with the the owner of the keyring file. Duplication here as well looks unnecessary.